source/docs/advanced/production-ready/front-commerce-and-pm2.md

```diff
@@ -1,12 +1,11 @@
 ---
 id: front-commerce-and-pm2
 title: Front-Commerce and PM2
+description: "Front-Commerce is a node.js application. You may want to run it in production with the PM2 Process Manager. This page explains how you could achieve this, and benefit from PM2 features such as: zero-downtime deployment, cluster mode, autoreload, logs aggregation…"
```

Comment:
Are the quotes around the description wanted?

Reply:
Yes. It is due to the `:` in the description.
Without quotes it would mess the YAML parsing.